shared/templates/create_auditd_lineinfile.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python2
+
+#
+# create_auditd_lineinfile.py
+#   automatically generate checks for auditd line in file configuration
+
+import sys
+
+from template_common import FilesGenerator, UnknownTargetError, CSVLineError
+
+
+class AuditdLineinfileGenerator(FilesGenerator):
```

Comment:
What about creating super class e.g. `LineinfileGenerator`, and these Auditd/sshd classes would inherit from it? Because the first part from `generate` (try/catch block) will be same for every lineinfile generator, right?

Reply:
That sounds interesting. There will be some differences but I think it is still worthy to have the super class. I'll think of something here.

Reply:
Great, thank you!

Reply:
I've updated the PR, please check it.